jetty-io/src/main/java/org/eclipse/jetty/io/MappedByteBufferPool.java

```diff
@@ -302,4 +308,30 @@ public String toString()
             getMaxBucketSize(),
             getCapacityFactor());
     }
+
+    protected class Retained extends ArrayRetainableByteBufferPool
```

Comment:
Seems strange to me that you would create a `MappedByteBufferPool` and ask it for its `RetainableBufferPool` and it gives you an array based implementation.

Reply:
yeah, but not sure we want to create a MappedRetainableByteBuffer pool.    Prior to the combination of pool, this is exactly what we would have got if the user selected a MBBP, as we would have created an ARBBP.   So just a bit strange that they are together.

I'm not actually sure if we really want to keep MBBP. @sbordet @lorban why do we have it?

Reply:
OK, but for this PR, I'd like to fix javadoc and MBBP to unblock other work.
We can consider deprecating MBBP later.

Reply:
Sounds good to me, and the change looks reasonable enough: currently MBBP is used together with a ARBBP so this change just solidifies a behavior that already exist.

Reply:
@lorban can I get a green tick then please?